bin/pycbc_banksim

```diff
@@ -374,7 +374,7 @@ with ctx:
                                       template_sample_rate,
                                       filter_N, options.filter_sample_rate)
 
-            h_norm = sigmasq(htilde, psd=psd, low_frequency_cutoff=f_lower,
+                h_norm = sigmasq(htilde, psd=psd, low_frequency_cutoff=f_lower,
```

Comment:
I don't see a problem with this change. I don't see any input to this function that is set within the loop over signals. The flower is set in the outer for loop over templates and set to None before looping over signals. What do you think is the issue here?

The problem with the old method is that sigmasq is computed for *every* signal+template combination, but its input only depends on template parameters. This is not an insignificant contribution to computational cost, so should not be calculated when not needed.